consensus/safety-rules/src/local.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) The Libra Core Contributors
```

Comment:
You have a single commit in which you moved a file but didn't explain why you did. That makes it really hard to review. Ideally you would move this in one commit and modify it in the next. Perhaps you could do that during the rebase?

Reply:
Alternatively, don't move this file.